runners/google-cloud-dataflow-java/worker/windmill/src/main/proto/windmill.proto

```diff
@@ -758,18 +758,15 @@ message WorkerMetadataResponse {
   // CommitWorkStream. Each response on this stream replaces the previous, and
   // connections to endpoints that are no longer present should be closed.
   message Endpoint {
-    optional string endpoint = 1;
+    // IPv6 address of a streaming engine windmill worker.
+    optional string direct_endpoint = 1;
+    optional string worker_token = 2;
   }
   repeated Endpoint work_endpoints = 2;
 
   // Maps from GlobalData tag to the endpoint that should be used for GetData
   // calls to retrieve that global data.
   map<string, Endpoint> global_data_endpoints = 3;
-
-  // DirectPath endpoints to be used by user workers for streaming engine jobs.
-  // DirectPath endpoints here are virtual IPv6 addresses of the windmill
-  // workers.
-  repeated Endpoint direct_path_endpoints = 4;
```

Comment:
reserve the tag # so it is not reused

reserved 4;

Reply:
done.

Reply:
I don't see a newer commit.  You're likely not done addressing the other comments yet but just fyi in case you're waiting on me.

Reply:
yep changes are still in the repo, was just using this to keep track of what i was finished with will tag you when finished thanks!